### ISSUE TITLE
Fix insecure warning from linking gjum's civmaps from cross origion

### ISSUE THREAD

**Issue by rourke750** [rourke750/SnitchTracker#9]
I don't know too much about website development, I believe that having a reference to a website that isnt on the same domain is causing issues.  Need to look further into it.

**Comment by Gjum** [comment]
can you show the exact warning you get? it's likely just the tiles being served over http now instead of https.

**Comment by rourke750** [comment]
Yep, you can just visit https://snitch.sandislandserv.com and you'll see the error on the homepage.  You need to be logged in though.

**Comment by Gjum** [comment]
Pretty sure this is just a warning and could be fixed by me setting up TLS, which is out of scope for me at this time.

**Comment by rourke750** [comment]
Ah sure, if you want its pretty simple.  Look into [certbot](https://certbot.eff.org/), its free and easy and you can have it set up in like 20 minutes.
